core_impl/src/main/java/io/opencensus/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2017, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Implementations that are internal to OpenCensus.
+ *
+ * <p>All the content under this package and its subpackages are considered annotated with {@link
+ * io.opencensus.common.Internal}.
+ */
+@io.opencensus.common.Internal
```

Comment:
sorry. yeah in OSGi, the default is that each package is sealed within a jar. You can use fragments to contribute other classes to a jar https://stackoverflow.com/questions/25670577/should-osgi-fragments-export-packages-contributed-to-their-host

There are other things about classloading which are simpler when everything in a package is in the same jar. If memory serves me right, package-private access is more reliable if in the same jar/classloader.